--- bootstrap.sh.bk	2023-06-29 11:08:33.008305471 -0400
+++ bootstrap.sh	2023-06-29 11:09:25.128867057 -0400
@@ -32,7 +32,7 @@
 	(
 		(
 			echo "🌏 Fetching server (this might take a while to finish)" &&
-				git clone https://github.com/nextcloud/server.git --depth 1 workspace/server 2>&1 | indent_cli &&
+				git clone https://github.com/nextcloud/server.git -b 'v27.0.0' --depth 1 workspace/server 2>&1 | indent_cli &&
 				cd workspace/server && git submodule update --init 2>&1 | indent_cli
 		) || echo "❌ Failed to clone Nextcloud server code"
 	) | indent
